roles/openshift_health_checker/openshift_checks/etcd_imagedata_size.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about an etcd cluster size based on OpenShift image data.
+"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdImageDataSize(OpenShiftCheck):
+    """Check that total size of OpenShift image data does not exceed the recommended limit in an etcd cluster"""
+
+    name = "etcd_imagedata_size"
+    tags = ["etcd"]
+
+    def run(self, tmp, task_vars):
+        ansible_mounts = get_var(task_vars, "ansible_mounts")
+
+        etcd_mountpath = self.get_etcd_mountpath(ansible_mounts)
+        etcd_avail_diskspace = etcd_mountpath["size_available"]
+        etcd_total_diskspace = etcd_mountpath["size_total"]
+
+        etcd_imagedata_size_limit = get_var(task_vars,
+                                            "cli_etcd_max_image_data_size_bytes",
```

Comment:
Guess it should be `r_openshift_health_checker_etcd_max_image_data_size_bytes`

Which something in the role should set from `openshift_health_etcd_image_data_bytes`

Whee...